.circleci/tests/nimbus-all.json

```diff
@@ -25,8 +25,5 @@
       "cl_client_type": "nimbus"
     }
   ],
-  "network_params": {
```

Comment:
if we are preserving capella fork epoch at all; lets keep a test

Reply:
added in a new test for capella.